spring-cloud-dataflow-server-core/src/test/java/org/springframework/cloud/dataflow/server/config/DefaultEnvironmentPostProcessorTests.java

```diff
@@ -16,46 +16,54 @@
 
 package org.springframework.cloud.dataflow.server.config;
 
-import org.junit.Test;
+import org.junit.jupiter.api.Test;
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.context.ConfigurableApplicationContext;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertNotNull;
+import static org.assertj.core.api.Assertions.assertThat;
 
 /**
  * Tests for {@link DefaultEnvironmentPostProcessor}.
  *
  * @author Josh Long
  * @auhor Chris Bono
  */
-public class DefaultEnvironmentPostProcessorTests {
+class DefaultEnvironmentPostProcessorTests {
 
-	private static final String MANAGEMENT_CONTEXT_PATH = "management.contextPath";
-
-	private static final String CONTRIBUTED_PATH = "/bar";
+	private static final String MANAGEMENT_CONTEXT_PATH = "management.context-path";
 
 	@Test
-	public void testDefaultsBeingContributedByServerModule() throws Exception {
+	void defaultsAreContributedByServerModule() {
 		try (ConfigurableApplicationContext ctx = SpringApplication.run(EmptyDefaultTestApplication.class, "--server.port=0",
 				"--spring.main.allow-bean-definition-overriding=true",
 				"--spring.autoconfigure.exclude=org.springframework.cloud.deployer.spi.cloudfoundry.CloudFoundryDeployerAutoConfiguration,org.springframework.cloud.deployer.spi.kubernetes.KubernetesAutoConfiguration")) {
-			String cp = ctx.getEnvironment().getProperty(MANAGEMENT_CONTEXT_PATH);
-			assertEquals(CONTRIBUTED_PATH, cp);
+
+			// this one comes from <test>/resources/dataflow-server.yml
+			assertThat(ctx.getEnvironment().getProperty(MANAGEMENT_CONTEXT_PATH)).isEqualTo("/foo");
```

Comment:
**[UNRELATED]** Having the value as constant was not helping readability (in fact the opposite). 

I also moved the default to be `/foo` and then the override to be `/bar` as the order in our head is "foo -> bar" rather than "bar -> foo".